src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,19 @@
-const numberToReversedDigits = (number) => {};
+const numberToReversedDigits = (number) => {
+  const reverse = String(number).split("").reverse();
+  return reverse.map(rev => Number(rev)); 
+
```

Comment:
Dont need the blank line here at the end of the function, should remove this. Only time blank lines are usually used inside functions is to separate things like assertions in tests, or to break up different internal parts to make it easier to read